docs/src/adoc/index.adoc

```diff
@@ -597,44 +602,6 @@ LocalDate releaseDate = handle.createQuery(
     .findOnly();
 ----
 
-==== LIKE clauses
-
-Due to a built-in safety mechanism, `LIKE` clauses and argument binding
```

Comment:
This isn't really a safety mechanism, it's just how jdbc works.  Parameters may not be bound into the middle of string literals (even if we decided to support the syntax, the driver won't do it.)